lib/constants/constants.go

```diff
@@ -54,3 +54,16 @@ const (
 	// Ops specifies a special cloud provider - a telekube Ops Center
 	Ops = "ops"
 )
+
+// Gravity API constants redeclared here to avoid extra dependencies solely to
+// get these values
+const (
+	// SiteState come from https://github.com/gravitational/gravity/blob/7.0.0/lib/ops/constants.go#L64-L93
+	SiteStateActive   = "active"
+	SiteStateDegraded = "degraded"
+
+	// SystemStatus come from https://github.com/gravitational/satellite/blob/7.1.0/agent/proto/agentpb/agent.pb.go#L28-L32
+	SystemStatus_Unknown  = 0
+	SystemStatus_Running  = 1
+	SystemStatus_Degraded = 2
+)
```

Comment:
Should these be in lib?  Seems like `infra/gravity/constants` or similar may be a better home.  I need to look into what the dividing line between `lib` and `infra` is intended to be.